api_backend.yaml

```diff
@@ -645,84 +645,150 @@ paths:
           description: Unavailable service
           schema:
             $ref: "#/definitions/ProblemJson"
+
+  "/user-data-processing":
+    x-swagger-router-controller: UserDataProcessingController
+    post:
+      operationId: upsertUserDataProcessing
+      summary: Set User's data processing choices
+      description: Upsert user data processing for the current authenticated user.
+      parameters:
+        - in: body
+          name: body
+          schema:
+            $ref: "#/definitions/UserDataProcessingChoiceRequest"
+          required: true
+      responses:
+        '200':
+          description: User Data processing created.
+          schema:
+            $ref: "#/definitions/UserDataProcessing"
+        "400":
+          description: Invalid payload.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+        "401":
+          description: Bearer token null or expired.
+        "409":
+          description: Conflict.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+        "500":
+          description: User Data processing choice cannot be taken in charge.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+  "/user-data-processing/{choice}":
+    x-swagger-router-controller: UserDataProcessingController
+    post:
+      operationId: getUserDataProcessing
+      summary: Get User's data processing 
+      description: Get user data processing for the current authenticated user and the given choice.
+      parameters:
+        - $ref: "#/parameters/UserDataProcessingChoiceParam"
+      responses:
+        "200":
+          description: User data processing retrieved
+          schema:
+            $ref: "#/definitions/UserDataProcessing"
+        "400":
+          description: Invalid request.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+        "401":
+          description: Bearer token null or expired.
+        "404":
+          description: Not found.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+        "429":
```

Comment:
it is not clear on why this may return 429 and upsertUserDataProcessing does not

Reply:
it's my fault, this may return 429 as GetUserDataProcessing ( IResponseErrorTooManyRequests is provided by the controller's method )